src/models/decoder_only_pipeline.cpp

```diff
@@ -13,6 +13,10 @@ DecoderOnlyPipelineModel::DecoderOnlyPipelineModel(std::unique_ptr<Config> confi
   for (const auto& model : config_->model.decoder.pipeline) {
     sessions_.emplace_back(OrtSession::Create(ort_env, (config_->config_path / fs::path(model.filename)).c_str(),
                                               GetSessionOptions(model.model_id)));
+  }
+
+#if 0
+  // TODO: Baiju, is this still needed?
```

Comment:
We probably don't need this anymore since we can assume the device allocator has already been created.